packages/documentation-site/patternfly-docs/content/design-tokens/design-tokens.md

```diff
@@ -0,0 +1,79 @@
+---
+id: About
+section: design-tokens
+---
+
+# Design tokens 
+
+This guide outlines PatternFly's use of design tokens. You should familiarize yourself with the details across this guide to learn about the purpose and implementation of PatternFly's tokens. 
+
+PatternFly's tokens are only available within Figma, so it's important that you follow our [Figma migration](/design-tokens/migrate) guide if necessary.
+
+## What are design tokens
+
+Design tokens are variables that store visual design attributes, like color, typography, and spacing. A design token has a name and value that represents the visual design information it contains. 
+
+## Why use tokens 
+
+Using design tokens helps maintain consistency across product design processes by ensuring that designers use the same colors for the same use cases. 
+
+Tokens also enable easier maintenance. Whenever a color is changed as part of a redesign or update,related tokens are automatically updated to reflect this change. This means that developers won't have to make any changes to code to ensure that their product is up to date with the latest recommendations. Likewise, designers will see that their designs automatically update to reflect any changes.
+
+## PatternFly design tokens
+
+### How to read tokens 
+
+Tokens can be read from left to right, with each part separated by a forward slash (/):
+
+`scope / component / category / property / concept / variant / state` 
+
+Each piece represents different information:
+- **Scope:** The range of the token. For example, `global` signifies that a token applies across the system.
+- **Component:** The component that the token relates to. 
+- **Category:** NEED INFO
```

Comment:
Here's a spreadsheet I made of the layers - we might actually not need some depending on how things shake out - e.g. I don't think we are really using "category." https://docs.google.com/spreadsheets/d/1fqNlufhJdm8V7xbGhxdSBYy_8N8riF3nXcvMXU1Z0K4/edit#gid=1732435221